modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/ContentContainer/DefinitionOfTerms.tsx

```diff
@@ -57,7 +57,7 @@ export function DefinitionOfTerms({
 			createResourceURL(baseResourceURL, {
 				objectDefinitionId: objectDefinition.id,
 				p_p_resource_id:
-					'/notification_templates/notification_template_terms',
+					'/notification_templates/notification_template_object_field',
 			}).toString()
```

Comment:
rename `getEntityFields` to `getObjectFieldTerms`

and

rename `setEntityFields` to `setObjectFieldTerms`